### ISSUE TITLE
Feed name on adding new feed

### ISSUE THREAD

**Issue by specter78** [brentsimmons/NetNewsWire#700]
Instead of showing `Name (optional)`, the app should rather fetch the feed name and allow user to edit it.

**Comment by mikehay** [comment]
Adding a vote for this issue. I had this same thought when I was adding several feeds today.

**Comment by brentsimmons** [comment]
The problem is that it needs to actually find the feed before it can fetch its name — and it doesn’t look for the feed until you click the button.

**Comment by mikehay** [comment]
Makes sense. Is there a way that the Safari extension (which is how I was adding feeds) could send the title of the page when it triggers the feed URI to NetNewsWire ?

**Comment by miklb** [comment]
asked about this in Slack. @danielpunkass explained there wasn't an option at the time to grab the `<title>` from the URL but might be something that is doable now.

**Comment by miklb** [comment]
Gruber made good point that the `<title>` might be the post/page title so should grab from the the feed enclosure.
